microshift_release_notes/microshift-4-17-release-notes.adoc

```diff
@@ -34,85 +34,76 @@ This release adds improvements related to the following components and concepts.
 
 [id="microshift-4-17-updating_{context}"]
 === Updating
-Updating two minor versions in a single step is now supported. Updates for both single-version minor releases and patch releases are still supported.
+//Updating two minor versions in a single step is supported from 4.16. Updates for both single-version minor releases and patch releases are still supported.
```

Comment:
```suggestion
//Updating two minor versions in a single step is supported in {what} 4.16 and higher. Updates for both single-version minor releases and patch releases are still supported.
```

Reply:
line 37 is commented. So the contents of this line are not visible in the preview. No changes are required in this line as it is commented.